web/impact/impact/v1/views/reserve_office_hour_view.py

```diff
@@ -256,7 +260,7 @@ def get_calendar_data(self, counterpart_name):
         location_info = location_info.format(location=location,
                                              separator=separator,
                                              meeting_info=meeting_info)
-        return Add2Cal(
+        self.calendar_data = Add2Cal(
```

Comment:
Typically you'd use a leading underscore to indicate that this field is in some sense "private" (ie, that one should not rely on it being there, and should call the get method instead) 

Not super critical, but it's useful to abide by these conventions